test/numbers-tests.js

```diff
@@ -275,19 +383,29 @@ describe('core-js-numbers', () => {
     () => {
       assert.equal(tasks.isInteger(5), true, 'value 5 is an integer');
       assert.equal(tasks.isInteger(5.1), false, 'value 5.1 is not an integer');
-      assert.equal(tasks.isInteger('5'), false, 'value 3/2 is not an integer');
+      assert.equal(tasks.isInteger('5'), false, 'value "5" is not an integer');
```

Comment:
Please make messages consistent. If you use double quotes around value in the last message, add such quotes to other messages.

Reply:
Thank you! Corrected.